app/build.gradle

```diff
@@ -93,6 +94,50 @@ android {
     }
 }
 
+tasks.register('codeCoverage', JacocoReport) {
+    dependsOn 'testDebugUnitTest'
+
+    reports {
+        html.required.set(true)
+        xml.required.set(true)
+    }
+
+    classDirectories.setFrom(
+            fileTree(project.buildDir) {
+                include("**/tmp/kotlin-classes/debug/**")
+                exclude(
```

Comment:
we can exclude some classes so that we they are not included in the code coverage report